packages/teams-js/src/public/app.ts

```diff
@@ -886,6 +898,18 @@ export namespace app {
     );
   }
 
+  /**
+   * Registers a function for handling data of hub to app message delay.
+   *
+   * @remarks
+   * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   *
+   * @param handler - The handler to invoke when the metrics are available on each function response.
+   */
+  export function registerHubToAppPerformanceMetricsHandler(handler: HubToAppPerformanceMetricsHandler): void {
+    Handlers.registerHubToAppPerformanceMetricsHandler(handler);
```

Comment:
Do you need "ensureInitialized(runtime)" as well like other handlers?

Reply:
In this instance the initialization is not necessarily a precondition for registering the handler, as it doesn't need to communicate with the host for registering the callback.